WhaleSpotting/Controllers/WhaleSightingController.cs

```diff
@@ -27,4 +28,18 @@ public IActionResult GetById([FromRoute] int Id)
             return NotFound();
         }
     }
+
+    [HttpGet("allpending")]
```

Comment:
Can we call this `pending` instead?
```suggestion
    [HttpGet("pending")]
```